cypress/integration/chi-custom-elements/alert.spec.js

```diff
@@ -98,11 +117,11 @@ describe('Alert', () => {
     it('Alert banner dismiss button should trigger appropriate event', () => {
       const spy = cy.spy();
 
-      cy.get('body').then((el) => {
+      cy.get('body').then(el => {
         el.on('dismissAlert', spy);
       });
 
-      cy.get('@alert-banner-large-success-dismiss')
+      cy.get('@alert-banner-dismissible-success-default')
```

Comment:
Sizes do not match here (replaced and new). Is this correct?

Reply:
Yes, it's correct. We added several tests / changed existing ones. That particular test checks if clicking chi-button inside of the alert triggers the appropriate event, size is irrelevant in this case.